proc_stat.go

```diff
@@ -152,14 +152,35 @@ func (s ProcStat) StartTime() (float64, error) {
 	if err != nil {
 		return 0, err
 	}
-	return float64(stat.BootTime) + (float64(s.Starttime) / ticks()), nil
+	return float64(stat.BootTime) + (float64(s.Starttime) / clkTck), nil
 }
 
 // CPUTime returns the total CPU user and system time in seconds.
 func (s ProcStat) CPUTime() float64 {
-	return float64(s.UTime+s.STime) / ticks()
+	return float64(s.UTime+s.STime) / clkTck
+}
+
+// clkTck is the CLK_TCK value usually set to 100Hz. While this could be read
+// from uninstd.h without the need to call an external command, the cgo
+// dependency prevents cross-compilation among others. On all tested systems
+// supporting procfs, getconf was available and returned the expected value.
+var clkTck = 100.
+
+func init() {
+	output, err := exec.Command("getconf", "CLK_TCK").Output()
```

Comment:
Looking at the glibc and linux source, it's 100 - except on ia64 and alpha. ia64 is configurable at kernel compile time (CONFIG_HZ) and alpha is 1024.